stock_picking_package_info/views/product_packaging_template_view.xml

```diff
@@ -0,0 +1,120 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+    Copyright (C) 2016-TODAY LasLabs, Inc. [https://laslabs.com]
```

Comment:
For the record, these files did not have a copyright on them previous. [See this link.](https://github.com/odoomrp/odoomrp-wip/blob/8.0/stock_picking_package_info/views/stock_quant_package_view.xml)